test/e2e/provisioning/all_e2e_test.go

```diff
@@ -624,7 +624,7 @@ func TestAzureRedhatSatelliteProvisioningE2E(t *testing.T) {
 		name:              "Azure redhat satellite server subscription",
 		osName:            "rhel",
 		containerRuntime:  "docker",
-		kubernetesVersion: "v1.17.0",
+		kubernetesVersion: "1.21.5",
```

Comment:
This is getting rid of the `v` prefix in `kubernetesVersion`, is that intentional? In this file, it's sometimes there for a scenario and sometimes isn't.

Reply:
I think for the versions I changed, I removed the prefix. That was IIRC the only thought that went into it.

Reply:
After doing some research, it seems that without the `v` prefix is correct. This is being templated into manifests for `MachineDeployments` [here](https://github.com/kubermatic/machine-controller/blob/a5b2d57b475da95de59368f20cfe936dac55d0a5/test/e2e/provisioning/helper.go#L143). In our examples, the kubelet version is given without the prefix.